resources/developer-mode.mjs

```diff
@@ -1,7 +1,10 @@
-import { Suites, Tags } from "./tests.mjs";
 import { params } from "./shared/params.mjs";
 
-export function createDeveloperModeContainer() {
+let suites, tags;
+
+export function createDeveloperModeContainer(_suites, _tags) {
```

Comment:
nit: usually prefixing with a `_` means that it's unused.

You could call them `aSuites` and `aTags`...